packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -257,9 +306,94 @@ class RenderParagraph extends RenderBox {
     return _textPainter.computeDistanceToActualBaseline(baseline);
   }
 
+  /// Intrinsics cannot be calculated without a full layout for
+  /// alignments that require the baseline.
+  bool _canComputeIntrinsics() {
+    for (PlaceholderSpan span in _placeholderSpans) {
+      if (span.alignment == ui.PlaceholderAlignment.baseline ||
+          span.alignment == ui.PlaceholderAlignment.aboveBaseline ||
+          span.alignment == ui.PlaceholderAlignment.belowBaseline) {
```

Comment:
please always use a switch when dealing with enums, not `==` comparisons. That way the analyzer will catch the case where we add a new value and forget to deal with it.